### DISCUSSION TITLE
Custom title not showing in generated openapi.json, it is always the default 'FastAPI'

### DISCUSSION THREAD

**Discussion by Daschi1** [fastapi/fastapi#9905] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI(titel="My Custom Title", version="1.0.0", redoc_url=None)
```

### Description

The custom title is not in the generated openapi json and always defaults to 'FastAPI'. The version field is correctly set.

* Open the browser and go to /docs
* The title is 'FastAPI' and not 'My Custom Title'
* When navigating to /openapi.json, the title is incorrectly set here too

### Operating System

Linux, Windows

### Operating System Details

I build the docker image on Linux, but access the URL via chrome on Windows.

### FastAPI Version

0.100.0

### Python Version

3.9

### Additional Context

This is the docker file I am using if that is maybe relevant:
```
FROM python:3.9
WORKDIR /app
COPY ./requirements.txt /app/requirements.txt
RUN pip install --no-cache-dir --upgrade -r /app/requirements.txt
COPY ./app /app/app
CMD ["uvicorn", "app.main:app", "--proxy-headers", "--host", "0.0.0.0", "--port", "80"]
```

**Comment by Daschi1** [comment]
I have found a workaround by setting app.title after initializing FastAPI:
```python
from fastapi import FastAPI

app = FastAPI(titel="My Custom Title", version="1.0.0", redoc_url=None)

app.title = "My Custom Title"
```

**Comment by jgould22** [comment, marked as answer]
If your example code is how your actual code is you have a typo
```
app = FastAPI(titel="My Custom Title", version="1.0.0", redoc_url=None)
                  ^
```

should be 

```
app = FastAPI(title="My Custom Title", version="1.0.0", redoc_url=None)
                  ^
```

**Reply by Daschi1** [reply]
I am feeling so dumb right now 🤦‍♂️
What the hell are IDEs for if they don't warn me that the parameter I am setting does not exist...
Anyway, thank you very much!
